File: Models/StundenKonfiguration.swift
import Foundation

struct StundeZeit: Identifiable {
    let id = UUID()
    let nummer: Int
    let von: String
    let bis: String
    let pauseDanach: Bool
}

struct StundenKonfiguration {
    static let alle: [StundeZeit] = [
        .init(nummer: 1, von: "07:30", bis: "08:15", pauseDanach: false),
        .init(nummer: 2, von: "08:15", bis: "09:00", pauseDanach: true),
        .init(nummer: 3, von: "09:15", bis: "10:00", pauseDanach: false),
        .init(nummer: 4, von: "10:00", bis: "10:45", pauseDanach: true),
        .init(nummer: 5, von: "11:05", bis: "11:50", pauseDanach: false),
        .init(nummer: 6, von: "11:50", bis: "12:35", pauseDanach: true),
        .init(nummer: 7, von: "13:15", bis: "14:00", pauseDanach: false),
        .init(nummer: 8, von: "14:00", bis: "14:45", pauseDanach: false),
        .init(nummer: 9, von: "14:45", bis: "15:30", pauseDanach: false)
    ]

    static let tage = ["MO", "DI", "MI", "DO", "FR"]
}

File: Windows-App/Unterrichtsplaner/Models/StundenKonfiguration.cs
namespace Unterrichtsplaner.Models;

public record StundeZeit(int Nummer, string Von, string Bis, bool PauseDanach);

public static class StundenKonfiguration
{
    public static readonly StundeZeit[] Alle =
    [
        new(1, "07:30", "08:15", false),
        new(2, "08:15", "09:00", true),
        new(3, "09:15", "10:00", false),
        new(4, "10:00", "10:45", true),
        new(5, "11:05", "11:50", false),
        new(6, "11:50", "12:35", true),
        new(7, "13:15", "14:00", false),
        new(8, "14:00", "14:45", false),
        new(9, "14:45", "15:30", false),
    ];

    public static readonly string[] Tage = ["MO", "DI", "MI", "DO", "FR"];
}
